isis/src/base/apps/reduce/tsts/outputLinesRoundOff/Makefile

```diff
@@ -8,7 +8,6 @@ APPNAME = reduce
 # user had entered.
 
 labels.txt.IGNORELINES = ByteOrder StartByte Bytes TileSamples TileLines
-results.pvl.SKIPLINES=2
```

Comment:
what 2 lines did we need to skip in the results.pvl before? (ie why did we remove this line)

Reply:
The output used to have % Processed, so it used to skip those lines. Now, the output does not have % Processed, so it was skipping the first two lines of the PVL.